service/README.md

```diff
@@ -0,0 +1,33 @@
+# Service lifecycle
+
+Service, being a bit of an overloaded term, requires some
+specific care to explain the various components in the `service`
+package in UberFx.
+
+## Instantiation
+
+Generally, you create a service in one of two ways:
+
+* The builder pattern, e.g. `service.WithModules(...).Build()`
```

Comment:
Quick fix: "The builder pattern (e.g., `service.WithModules(…).Build()`)"

Longer fix: Is this the way everyone always uses the builder pattern? It seems like "in this case" or "that is" might be less confusing to readers than "e.g."